azavea/pfb-network-connectivity
# Configure tile URL based on settings

## Overview

Select either new Tilegarden endpoint URL if `TILEGARDEN_ROOT` has been defined on the environment, or fall back to previous URL structure.


### Notes

We will want to either revert this PR once the intermediate production deployment it's here to support is done, or re-set the production `tilegarden_root` variables in `app.tf` when we're ready to switch production to use Tilegarden.


## Testing Instructions

 * Should still load Tilegarden tiles
 * Remove `TILEGARDEN_ROOT` passed env var from `common.yml`, or set to an empty string
 * Restart `./scripts/server`
 * Tiles should be pulled from S3 (unless the last neighborhood job was a re-import, which won't have any)


Closes #641.
